app/services/git_based_domain_import_service.rb

```diff
@@ -62,6 +81,16 @@ def import(git_repo_id, branch_or_tag, tenant_id)
     domain.update_attribute(:enabled, true)
   end
 
+  def refresh(git_repo_id)
```

Comment:
@gmcculloug We can refactor this in a different PR since it affects other areas of the product REST API and UI specifically